StorageUtility/Linux/Device.h

```diff
@@ -33,29 +29,24 @@ namespace vtStor
 class cDevice : public cDeviceInterface
 {
 public:
-//    cDevice(const PSP_DEVINFO_DATA& DevInfoData, const PSP_DEVICE_INTERFACE_DATA& DevInterfaceData, const PSP_INTERFACE_DEVICE_DETAIL_DATA& DevDetailData, U32 SizeOfDevDetailData);
-//    virtual ~cDevice();
+    cDevice();
+    cDevice (String Path);
+    virtual ~cDevice();
 
 public:
-  // virtual void Data(std::unordered_map<eDeviceDataType, void*>& Data) override;
-
-//   virtual DeviceHandle Handle() override;
-
-private:
-   /* eErrorCode GetStorageDeviceHandle(const String& DevicePath, HANDLE& Handle);
-
-
-    void AllocateMemories(const PSP_DEVINFO_DATA& DevInfoData, const PSP_DEVICE_INTERFACE_DATA& DevInterfaceData, const PSP_INTERFACE_DEVICE_DETAIL_DATA& DevDetailData, U32 SizeOfDevDetailData);
-
-    void CopyMemories(const PSP_DEVINFO_DATA& DevInfoData, const PSP_DEVICE_INTERFACE_DATA& DevInterfaceData, const PSP_INTERFACE_DEVICE_DETAIL_DATA& DevDetailData, U32 SizeOfDevDetailData);
+    virtual void Data(std::unordered_map<eDeviceDataType, void*>& Data) override;
+    virtual DeviceHandle Handle() override;
+    virtual void DevicePath(tchar*& DevicePath) override;
 
-    void DeallocateMemories();*/
+    String GetDevicePath();

```

Comment:
Why define more function get and set for DevicePath and DeviceHandle?
Don't we have the Handle() and DevicePath(tchar*& ...) already